partiql-conformance-tests/tests/mod.rs

```diff
@@ -30,6 +30,33 @@ pub(crate) fn pass_syntax(statement: &str) {
     );
 }
 
+#[track_caller]
+#[inline]
+pub(crate) fn fail_semantics(_statement: &str) {
+    todo!("fail_semantics")
+}
+
+#[track_caller]
+#[inline]
+#[allow(dead_code)]
+pub(crate) fn pass_semantics(_statement: &str) {
+    todo!("pass_semantics")
+}
+
+#[track_caller]
+#[inline]
+#[allow(dead_code)]
+pub(crate) fn fail_eval(_statement: &str) {
+    todo!("fail_semantics")
+}
+
+#[track_caller]
+#[inline]
+#[allow(dead_code)]
+pub(crate) fn pass_eval(_statement: &str) {
+    todo!("pass_semantics")
```

Comment:
nit: (similar to above comment)

```suggestion
    todo!("pass_eval")
```